app/v1/controllers/RetrieveCharitableGivingReliefController.scala

```diff
@@ -85,8 +85,16 @@ class RetrieveCharitableGivingReliefController @Inject() (val authService: Enrol
 
   private def errorResult(errorWrapper: ErrorWrapper) = {
     errorWrapper.error match {
-      case BadRequestError | NinoFormatError | TaxYearFormatError | RuleTaxYearNotSupportedError | RuleTaxYearRangeInvalidError =>
+      case _
+        if errorWrapper.containsAnyOf(
```

Comment:
Can you auto-format this class (+ the others, to make sure) with scalafmt?  It's needed so that the PRs are all consistent and don't end up with lines being reformatted back & forth by different devs. Thanks!

Reply:
If you set your Intellij settings to "format on save" (and make sure it's using scalafmt rather than Intellij's own code formatter), that should cover it for future PRs too